zombienet-tests/src/0002-parachain.zndsl

```diff
@@ -6,4 +6,4 @@ alice: is up
 bob: is up
 alice: parachain 100 is registered within 225 seconds
 alice: parachain 100 block height is at least 3 within 200 seconds
-bob: ts-script ./0002-checkSync.ts within 140 seconds
```

Comment:
not sure why i had to change this but it _seems_ to just work.

Times out randomly:
https://github.com/paritytech/substrate-connect/actions/runs/9306397328/job/25615572039?pr=2200

Reply:
https://github.com/paritytech/substrate-connect/pull/2212